packages/notion-utils/src/get-block-collection-pointer-id.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  CollectionViewBlock,
+  CollectionViewPageBlock,
+  ExtendedRecordMap
+} from 'notion-types'
+
+export function getBlockCollectionPointerId(
```

Comment:
i think this logic could be inside of `getBlockCollectionId` instead of exporting a new method. thoughts?

Reply:
So just have an if statement in there based on if the recordmap is passed in? Either way seems fine to me.

Reply:
Yep, that looks good!